src/ml/neural_net/mps_layer_helper.h

```diff
@@ -0,0 +1,44 @@
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h>
+#import <Metal/Metal.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+API_AVAILABLE(macos(10.14))  
+@interface MPSCNNFullyConnectedNode (TCMPSLayerHelper)
++ (MPSCNNFullyConnectedNode *) createFullyConnected:(MPSNNImageNode *)inputNode
+                               inputFeatureChannels:(NSUInteger)inputFeatureChannels
+                              outputFeatureChannels:(NSUInteger)outputFeatureChannels
+                                        inputHeight:(NSUInteger)inputHeight
+                                         inputWidth:(NSUInteger)inputWidth
+                                            weights:(float *)weights
+                                             biases:(float *)biases
+                                              label:(NSString *)label
+                                      updateWeights:(BOOL)updateWeights
+                                             device:(id<MTLDevice>)dev
+                                          cmd_queue:(id<MTLCommandQueue>)cmd_q
+                                          API_AVAILABLE(macosx(10.14));
```

Comment:
Are these `API_AVAILABLE(macosx(10.14))` required on each method, even when the overall interface has an `API_AVAILABLE` on it?

Reply:
Good catch! We don't need it in both places.